README.md

```diff
@@ -69,10 +69,13 @@ COMMANDS:
 GLOBAL OPTIONS:
    --file value, -f value       Vendored packages list (default: "vendor.conf")
    --directory value, -C value  The directory in which to run, --file is relative to this (default: ".")
+   --target value, -T value     The directory to store results (default: "vendor")
    --keep, -k                   Keep all downloaded vendor code (preserving .git dirs)
-   --update, -u                 Update vendored packages, add missing ones
+   --update value, -u value     specify a list of packages to be updated
```

Comment:
Would be nice to see all sentences start with a capital letter

Reply:
This is the output of `trash -h`. I didn't change any capitalisation, that would be outside of the scope of this PR.